os/kernel/sched/sched_cpuselect.c

```diff
@@ -92,7 +92,7 @@ int sched_select_cpu(cpu_set_t affinity)
               DEBUGASSERT(rtcb->sched_priority == 0);
               return i;
             }
-          else if (rtcb->sched_priority < minprio)
+          else if (rtcb->sched_priority <= minprio)
```

Comment:
Is this necessary checking? Is there any change to be greater than minprio `(rtcb->sched_priority > minprio)`?